### DISCUSSION TITLE
Can't resolve 'theme-ui/jsx-runtime'

### DISCUSSION THREAD

**Discussion by bajpai244** [system-ui/theme-ui#1379] [Q&A] [answered]
Hello Everyone, I am using a custom .babelrc in next js to support the automatic runtime.

```
{
  "plugins": [
    [
      "@babel/plugin-transform-react-jsx",
      {
        "throwIfNamespace": false, // defaults to true
        "runtime": "automatic", // defaults to classic
        "importSource": "theme-ui" // defaults to react
      }
    ]
  ]
}


```
I am getting an error which is **"Can't resolve 'theme-ui/jsx-runtime'"**.

Am I doing something wrong? As a fallback, I am using "classic runtime". I would appreciate some help here.

Some info about my dev environment:

     @babel/core:  ^7.12.10,
     @babel/plugin-transform-react-jsx:  ^7.12.11,
      next:  10.0.3,
      theme-ui: ^0.3.4

**Comment by lachlanjc** [comment, marked as answer]
Hey! The latest stable version doesn't support the automatic runtime—we're working on a new stable release soon, but for now I recommend installing `^0.6.0-alpha.3`. There are some other breaking changes (like we renamed `Styled` to `Themed`) but you can check out the `CHANGELOG`. Let me know if that works!

**Comment by bajpai244** [comment]
Yes! That works! thanks a lot for the help.
